tests/ops/integration_tests/saas/test_hubspot_task.py

```diff
@@ -91,8 +91,6 @@ async def test_hubspot_access_request_task(
     }
     assert set(filtered_results[f"{dataset_name}:contacts"][0].keys()) == {
         "id",
-        "createdAt",
```

Comment:
We don't this this for all of the connector tests but for this one we have step to verify that the fields with a data category of `user.*` are present after the policy filters the fields. Since `createdAt` and `updatedAt` are no longer considered user fields, these had to be removed.